src/js_ast.zig

```diff
@@ -6915,6 +6915,12 @@ pub const Ast = struct {
     }
 };
 
+pub const tlaCheck = struct {
```

Comment:
the standard casing convention in zig is pascal for struct names, and snake case for field names. `TlaCheck` and `import_record_index`.